index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <script async src="https://www.googletagmanager.com/gtag/js?id=G-WK01NEHZ12"></script>
+        <script>
+        window.dataLayer = window.dataLayer || [];
+        function gtag(){window.dataLayer.push(arguments);}
+        gtag('js', new Date());
+
+        </script>
+        <meta charset="utf-8">
+        <title> Carlos Philippe </title>
+        <link rel="stylesheet" type="text/css" href="style.css">
+        <link rel="preconnect" href="https://fonts.googleapis.com">
+        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+        <link href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,300;1,100;1,600&display=swap" rel="stylesheet">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+    </head>
+    <body>
+      <div class="home-page">
+        <header>
+          <nav id="nav">
+            <a class="logo" href="#"><img src="images/logo.png" alt="my logo" ></a>
+  <nav>
+    <class="hamburger"><img src="images/Union.png" alt="Menu">
+  </nav>
```

Comment:
- [ ] I noticed that you had made changes to the `logo` but couldn't position the hamburger button to the right. I would recommend that you apply these styles to the overall container that contains the `logo` and `hamburger` button
- [ ] ` display: flex `
- [ ]  `justify-content: space between`